### PR TITLE
clean up code a little

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -10,9 +10,8 @@ const Resizer = ({ array, updateCallback }) => {
       const newCells = Array(n - array[0].length).fill(0);
       array = array.map((r) => r.concat(newCells));
     }
-    if (n < array[0].length) {
-      array = array.map((r) => r.slice(0, n - array[0].length));
-    }
+
```
Comment (LukeStorry) [inline index.js:13]: This line is too long, so a linter will break it up over two lines anyway.
Once an `if` statement is more than one line, it's easier to read (and easier to change in the future) if the brackets are there.

Comment (aimorris) [reply index.js:13]: Ah alright

File: index.js [hunk 2]
```diff
@@ -10,9 +10,8 @@ const Resizer = ({ array, updateCallback }) => {
...
+    if (n < array[0].length) array = array.map((r) => r.slice(0, n - array[0].length));
```
Comment (LukeStorry) [inline index.js]: This line is too long, so prettier (& most linters) will break it up over two lines anyway.
Once an `if` statement is more than one line, it's easier to read (and easier to change in the future) if the brackets are there.

File: index.js [hunk 3]
```diff
@@ -60,7 +58,7 @@ const Resizer = ({ array, updateCallback }) => {
 
 const Input = ({ array, updateCallback }) => {
   const updateCell = (rowIndex, colIndex) => () => {
-    array[rowIndex][colIndex] = array[rowIndex][colIndex] ? 0 : 1;
+    array[rowIndex][colIndex] ^= 1;
```
Comment (LukeStorry) [inline index.js:65]: I like this a lot :)

**Review by LukeStorry — CHANGES_REQUESTED** [review]
If you remove the changes to the if statements, I'd like to just accept the ternary ^= line, it is very slick and I hadn't seen it before

**Comment by LukeStorry** [comment]
Nice!
You completed my requests before I'd even finished typing them! 
Looks good, will merge. :smile:

**Review by LukeStorry — APPROVED** [review]

**Pull request merged**